src/app/Sidebar/UserInfo.js

```diff
@@ -57,6 +57,14 @@ const UserInfo = ({ intl, authenticated, authenticatedUser, user, ...props }) =>
               }}
             />
           </div>
+          <div>
+            <i className="iconfont icon-flashlight text-icon" />
+            <FormattedMessage id="voting_power" defaultMessage="Voting Power" />: <FormattedNumber
+              style="percent" // eslint-disable-line react/style-prop-object
+              value={user.voting_power / 10000}
```

Comment:
So to get the percent of something you divide it by the total... so is 10,000 the total voting power that a user can have? @Sekhmet @bonustrack might know the answer

Reply:
@jm90m 
Steem uses 0-10000 range (10000 being 100%). `react-intl` uses 0-1 range, thus we need to divide it by 10000.

Reply:
Ah ok.... so `style="percent"` converts 0-1 to 0-100? Can't find this conversion in the code.

Reply:
It just tells `react-intl` to render this as per cent.
https://github.com/yahoo/react-intl/wiki/Components#formattednumber